src/Raven.Server/Documents/Indexes/IndexStorage.cs

```diff
@@ -734,6 +735,13 @@ public void WriteReferencesForSingleCollection(string collection, Dictionary<Sli
             }
         }
 
+        public long ReadLastProcessedTombstoneEtag(IndexItemType itemType, RavenTransaction tx, string collection)
+        {
```

Comment:
move that out of the method via abstract

Reply:
done